apps/api/v2/src/ee/calendars/services/calendars.service.ts

```diff
@@ -187,6 +205,27 @@ export class CalendarsService {
       userId,
       calendarType
     );
+
+    await this.deleteCalendarCache(userId);
+  }
+
+  async deleteCalendarCache(userId: number) {
+    await this.redisService.del(REDIS_CALENDARS_CACHE_KEY(userId));
+  }
+
+  async getCalendarCache(userId: number) {
```

Comment:
I would rename it `getConectedAndDestinationCalendarsCache` and also have `setConnectedAndDestinationCalendarsCache` because based on the awaited return type it's not clear what exactly is in the cache. It's a bit longer name but would inform what's cached + caching is called in destination and connected caledars services.